src/services/lookup-table-handler.ts

```diff
@@ -223,7 +220,7 @@ export const validateLookupTable = async (
   protoLookupTable: DataTable,
   dataset: Dataset,
   dimension: Dimension,
-  buffer: Buffer,
+  file: Express.Multer.File,
```

Comment:
If we're only using `file.path` then probably just pass the path string - if we're using multiple props e.g. path/buffer/origfilename etc then pass the file object.

Reply:
Changed this to be a string